alby-nostr-wallet-connect/umbrel-app.yml

```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: alby-nostr-wallet-connect
+name: Nostr Wallet Connect
+tagline: The power of the zaps at the tip of your fingers.
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
```

Comment:
Now that this will be in our app store, we no longer need to reference the icon here

```suggestion
```